src/protocols/bdx/TransferFacilitator.cpp

```diff
@@ -53,7 +53,7 @@ CHIP_ERROR TransferFacilitator::OnMessageReceived(chip::Messaging::ExchangeConte
     // transfer could necessitate a response if they are received at the wrong time.
     // For this reason, it is left up to the application logic to call ExchangeContext::Close() when it has determined that the
     // transfer is finished.
-    mExchangeCtx->WillSendMessage();
+    mExchangeCtx->WillSendMessage(true /* IsPeerActive */);
```

Comment:
This is a breaking public API change. WillSendMessage without arguments has to behave like it's always "true", otherwise any users of BDX not using this TransferFacilitator (e.g. Google), will not magically have the `true` argument.

Reply:
resolving since the this changes doesn't exist anymore.